src/Illuminate/Routing/Router.php

```diff
@@ -406,12 +409,12 @@ public static function mergeGroup($new, $old)
     protected static function formatUsesPrefix($new, $old)
     {
         if (isset($new['namespace']) && isset($old['namespace'])) {
-            return trim(array_get($old, 'namespace'), '\\').'\\'.trim($new['namespace'], '\\');
+            return trim($old['namespace'], '\\').'\\'.trim($new['namespace'], '\\');
```

Comment:
The condition already check for `isset()`, `array_get()` here is just redundant.